app/vmalert/config/testdata/rules/rules-multi-doc-good.rules

```diff
@@ -0,0 +1,39 @@
+
+---
+groups:
+  - name: groupTest
+    rules:
+      - alert: VMRows
+        for: 1ms
+        expr: vm_rows > 0
+        labels:
+          label: bar
+          host: "{{ $labels.instance }}"
+        annotations:
+          summary: "{{ $value }}"
+---
+groups:
```

Comment:
Could you also add a test for annotation with possible `---`:
```
        annotations:
          summary: \n markdown result is : ----\n # header\n body: \n text \n----\n"
```

Reply:
Good catch & I can get your point. Just updated the test cases, it seems the code still working.